js/schema/style.json

```diff
@@ -29,6 +29,9 @@
     "size": {
       "type": "string",
       "enum": ["small", "medium", "large"]
+    },
+    "height": {
```

Comment:
Hey! If we are going to add a pixel specific `height` to the overall `json-schema`, instead of just using the existing `size` prop (small, medium, and large), I think we need to consider how this will be rendered on other platforms like mobile apps/SDK, Facebook Messages, WhatsApp, etc.

I think maybe we might consider adding a sub-element like `web` to the style element on the json-schema, to define specific styles for web. We are already doing something similar for accessibility attributes that only make sense on web.

The probably is that setting a height of 20 CSS pixels on web may mean something completely different on mobile SDK for example. Let me know what you think.